migrations/20200514135000-add-connected-github-account-for-tests.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+module.exports = {
+  up: async queryInterface => {
+    await queryInterface.sequelize.query(`
+	INSERT INTO "ConnectedAccounts" (id, "CreatedByUserId", service, "clientId", username, token, "CollectiveId", "createdAt", "updatedAt")
+	VALUES (2136, 9475, 'github', 90020713, 'testuseradmingithub', 'foofoo', 10883, '2020-05-14 13:11:27.893+01', '2020-05-14 13:11:27.903+01');
+    `);
+  },
+
+  down: async queryInterface => {
+    await queryInterface.sequelize.query(`
+	DELETE FROM "ConnectedAccounts"
+	WHERE "id" = 2136;
+  `);
+  },
+};
```

Comment:
I'm not sure to follow why that's still necessary with the stub approach.

If you confirm it is:
- only insert in 'development', 'ci', and 'e2' environments
- remove the" id" from the insert, the db will simply autoincrement
- remove the "down" migration.

Reply:
You're completely right, this is leftover from my first approach. I'll remove it.

Reply:
Ok turns out we do need it, I will make sure it is only added in e2e and CI environments and make the other related changes.